contracts/seed/SeedFactory.sol

```diff
@@ -95,10 +95,10 @@ contract SeedFactory is CloneFactory {
     function deploySeed(
         address          _admin,
         address[] memory _tokens,
-        uint[]    memory _softAndHardCap,
-        uint  	         _price,
-        uint 	         _startTime,
-        uint 	         _endTime,
+        uint256[]    memory _softAndHardCap,
```

Comment:
pls change indent to spaces

Reply:
Updated with spaces. Also how do I find if it is a tab or space?